### ISSUE TITLE
when I run demo.py..I get following errors.. I have python 3.5 and tensor flow installed.. Thank you in advance

### ISSUE THREAD

**Issue by amitadhiman** [MarvinTeichmann/KittiSeg#26]
2017-03-27 00:10:23,039 INFO No environment variable 'TV_PLUGIN_DIR' found. Set to 'C:\Users\Amita/tv-plugins'.
2017-03-27 00:10:23,039 INFO No environment variable 'TV_STEP_SHOW' found. Set to '50'.
2017-03-27 00:10:23,054 INFO No environment variable 'TV_STEP_EVAL' found. Set to '250'.
2017-03-27 00:10:23,054 INFO No environment variable 'TV_STEP_WRITE' found. Set to '1000'.
2017-03-27 00:10:23,054 INFO No environment variable 'TV_MAX_KEEP' found. Set to '10'.
2017-03-27 00:10:23,054 INFO No environment variable 'TV_STEP_STR' found. Set to 'Step {step}/{total_steps}: loss = {loss_value:.2f}; lr = {lr_value:.2e}; {sec_per_batch:.3f} sec (per Batch); {examples_per_sec:.1f} imgs/sec'.
2017-03-27 00:10:23,070 ERROR No input_image was given.
2017-03-27 00:10:23,070 INFO Usage: python demo.py --input_image data/test.png [--output_image output_image] [--logdir /path/to/weights] [--gpus GPUs_to_use]

**Comment by MarvinTeichmann** [comment]
Run `python demo.py --input_image data/test.png`. Also, try using python 2.7.
